specification/purview/data-plane/Azure.Analytics.Purview.DataMap/stable/2023-09-01/purviewdatamap.json

```diff
@@ -0,0 +1,7267 @@
+            }
+          },
+          {
+            "$ref": "#/parameters/purviewAPIVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. On successful lookup of the suggestions.",
+            "schema": {
+              "$ref": "#/definitions/SuggestResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/search/autocomplete": {
+      "post": {
+        "tags": [
+          "Discovery"
+        ],
+        "description": "Get auto complete options.",
+        "operationId": "Discovery_AutoComplete",
+        "x-ms-examples": {
+          "Discovery_AutoComplete": {
+            "$ref": "./examples/Discovery_AutoComplete.json"
+          }
+        },
+        "parameters": [
+          {
+            "in": "body",
+            "name": "autoCompleteRequest",
+            "description": "An object specifying the autocomplete criteria.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AutoCompleteRequest"
+            }
+          },
+          {
+            "$ref": "#/parameters/purviewAPIVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. On successful lookup of the auto complete options.",
+            "schema": {
+              "$ref": "#/definitions/AutoCompleteResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/atlas/v2/lineage/{guid}": {
+      "get": {
+        "tags": [
+          "Lineage"
+        ],
+        "description": "Get lineage info of the entity specified by GUID.",
+        "operationId": "Lineage_GetGraph",
+        "x-ms-examples": {
+          "Lineage_GetGraph": {
+            "$ref": "./examples/Lineage_GetGraph.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/guid"
+          },
+          {
+            "$ref": "#/parameters/depth"
+          },
+          {
+            "$ref": "#/parameters/direction"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. If the request is valid.",
+            "schema": {
+              "$ref": "#/definitions/AtlasLineageInfo"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/lineage/{guid}/next/": {
```

Comment:
I am not sure about the purpose of this API. Is it a "List" or only the "{nextPage}" URL of another API?

Reply:
As the lineage info might contains too many entities, so this api is to get lineage with pagination support.